kombu/transport/redis.py

```diff
@@ -757,7 +767,8 @@ def _size(self, queue):
 
     def _q_for_pri(self, queue, pri):
         pri = self.priority(pri)
-        return '%s%s%s' % ((queue, self.sep, pri) if pri else (queue, '', ''))
+        key = '%s%s%s' % ((queue, self.sep, pri) if pri else (queue, '', ''))
+        return self.global_key_prefix + key
```

Comment:
I think this is redundant, doesn't the queue names (`queue` variable here) already contain `self.keyprefix_queue`, which in its turn is prefixed with the global namespace?

Reply:
@georgepsarakis I was puzzled by this too!

I assume that because this method comes from the base interface, it is just called with the bare queue name without prefix (because prefixes are introduced by the redis implementation).
https://github.com/celery/kombu/blob/0af7519f98453bae62e0f4ad70e613e04c8fa379/kombu/transport/virtual/base.py#L361

But I might be wrong! If this method is indeed called with the prefixed queue name, then I guess the second test case becomes trivial…